src/api-helpers.ts

```diff
@@ -108,6 +115,42 @@ export namespace ApiHelpers {
         return apiItem;
     }
 
+    export function PathIsInside(thePath: string, potentialParent: string): boolean {
```

Comment:
I think we could move this to `utils` in a separate file?

Reply:
FIxed.